cmd/helm/root_test.go

```diff
@@ -40,20 +40,20 @@ func TestRootCmd(t *testing.T) {
 		{
 			name:      "with $XDG_CACHE_HOME set",
 			args:      "home",
-			envars:    map[string]string{xdg.CacheHomeEnvVar: "/bar"},
-			cachePath: "/bar/helm",
+			envars:    map[string]string{xdg.CacheHomeEnvVar: os.TempDir() + "bar"},
+			cachePath: os.TempDir() + "bar/helm",
```

Comment:
Instead of this can you create a wrapper function which accomplishes this by returned the new path ?
Similar to
```
func lazyCreateDir(name string ){
return os.TempDir() + name
}
```